app-kotlin/src/main/kotlin/au/com/gridstone/trainingkotlin/data/DataModel.kt

```diff
@@ -0,0 +1,17 @@
+package au.com.gridstone.trainingkotlin.data
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+data class PokemonResults(val results: List<Pokemon>)
+
+@Parcelize
+data class Pokemon(
+  val url: String,
+  val name: String,
+  val id: String = url.split("/").dropLast(1).last(),
+) : Parcelable
+
+data class PokemonData(val stats: List<Stats>)
```

Comment:
is this used?

Reply:
no in this pr  it is used by the next pr, sorry about that i am getting used to this now will keep this inn mind next time.